StandaloneMmPkg/Library/StandaloneMmHobLib/StandaloneMmHobLib.c

```diff
@@ -318,6 +318,13 @@ BuildModuleHob (
     );
 
   Hob = CreateHob (EFI_HOB_TYPE_MEMORY_ALLOCATION, sizeof (EFI_HOB_MEMORY_ALLOCATION_MODULE));
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (Hob == NULL) {
+    ASSERT (Hob != NULL);

```

Comment:
nit: would it be easier to just `ASSERT (FALSE);`?

Reply:
The assert is not as obvious in that case.

- `ASSERT (FALSE)` Example:
  - `ASSERT Platforms\QemuQ35Pkg\Library\QemuFwCfgLib\QemuFwCfgLib.c(37): ((BOOLEAN)(0==1))`

- `ASSERT (Pointer != NULL)` Example:
  - `ASSERT Platforms\QemuQ35Pkg\Library\QemuFwCfgLib\QemuFwCfgLib.c(37): Ptr != ((void *) 0)`

Reply:
Going to close for now